src/main/java/slimeknights/tconstruct/smeltery/block/BlockSearedFurnaceController.java

```diff
@@ -0,0 +1,175 @@
+package slimeknights.tconstruct.smeltery.block;
+
+import java.util.Random;
+
+import javax.annotation.Nonnull;
+
+import net.minecraft.block.BlockHorizontal;
+import net.minecraft.block.SoundType;
+import net.minecraft.block.material.Material;
+import net.minecraft.block.properties.PropertyBool;
+import net.minecraft.block.properties.PropertyDirection;
+import net.minecraft.block.state.BlockStateContainer;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.entity.EntityLivingBase;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.item.ItemStack;
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.util.EnumBlockRenderType;
+import net.minecraft.util.EnumFacing;
+import net.minecraft.util.EnumParticleTypes;
+import net.minecraft.util.Mirror;
+import net.minecraft.util.Rotation;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.IBlockAccess;
+import net.minecraft.world.World;
+import slimeknights.tconstruct.common.block.BlockInventoryTinkers;
+import slimeknights.tconstruct.library.TinkerRegistry;
+import slimeknights.tconstruct.smeltery.tileentity.TileSearedFurnace;
+
+public class BlockSearedFurnaceController extends BlockInventoryTinkers {
+
+  public static final PropertyDirection FACING = BlockHorizontal.FACING;
+  public static final PropertyBool ACTIVE = PropertyBool.create("active");
+  
+  public BlockSearedFurnaceController() {
+    super(Material.ROCK);
+    this.setCreativeTab(TinkerRegistry.tabSmeltery);
+    this.setHardness(3F);
+    this.setResistance(20F);
+    this.setSoundType(SoundType.METAL);
+
+    this.setDefaultState(this.blockState.getBaseState().withProperty(FACING, EnumFacing.NORTH).withProperty(ACTIVE, false));
+  }
+
+  @Override
+  public TileEntity createNewTileEntity(World worldIn, int meta) {
```

Comment:
Hmm, not sure. I'll have to look into that.

I fix those up when I add a few of the other fixed.